.gitignore

```diff
@@ -0,0 +1,6 @@
+*.egg-info
```

Comment:
I tend to think files IDEs and editors create don't belong here, unless they are produced by tools virtually everyone working with this repo is using.  For other tools, it's up to the people using these tools to put adequate patterns in `.git/info/exclude` or `~/.config/git/ignore`.